tests/eas/cpuhp.py

```diff
@@ -77,18 +61,19 @@ def setUpClass(cls, *args, **kwargs):
     @classmethod
     def _getExperimentsConf(cls, test_env):
```

Comment:
Since we're overriding `runExperiments`, do we still need this?

Reply:
Nope, that's a remnant of previous version of this test, I'll remove it